lighthouse-core/test/lib/proto-preprocessor-test.js

```diff
@@ -27,6 +27,30 @@ describe('processing for proto', () => {
     expect(JSON.parse(output)).toMatchObject(expectation);
   });
 
+  it('cleans up default runtimeErrors', () => {
+    const input = {
+      'runtimeError': {
+        'code': 'NO_ERROR',
+      },
+    };
+
+    const output = processForProto(JSON.stringify(input));
+
+    expect({}).toMatchObject(JSON.parse(output));
+  });
+
+  it('ignores non-default runtimeErrors', () => {
```

Comment:
nit: `ignores` was a bit ambiguous to me as it could've meant "drops" WDYT about `passes through`, `leaves untouched`, `preserves`?